File: abc/abc209/a.cpp
// https://atcoder.jp/contests/abc209/submissions/25511725

#include <bits/stdc++.h>

using namespace std;

int main() {
  int A, B;
  cin >> A >> B;
  if(A > B) {
    cout << 0;
  } else {
    cout << B - A + 1;
  }
}

File: abc/abc209/a.rs
// https://atcoder.jp/contests/abc209/submissions/47525256

use proconio::input;

fn main() {
    input! {
        a: u8,
        b: u8
    }

    let mut cnt = 0;
    for i in a..(b + 1) {
       cnt += 1; 
    }

    print!("{}", cnt);
}

